README.md

```diff
@@ -461,7 +461,7 @@ leverages the system logrotate functionality to rotate logs.
 #### __Logs.Collect__
 
 _Type: Boolean_  
-_Default: n_
+_Default: y_
 
 If set, agent logs will be periodically collected and uploaded to a secure location for improved supportability.
```

Comment:
If I remembered correctly log collector enabled only when other two conditions met along with this. Is it worthwhile to mention here otherwise it confuse the customers, right?

Reply:
+1

Reply:
I think this is a valid point; I'll submit a rewrite, let me know what you think.

Reply:
Looks good.